minst/model.py

```diff
@@ -1,15 +1,24 @@
-
+import copy
 import json
 import jsonschema
+import logging
 import pandas as pd
 import os
+from sklearn.cross_validation import train_test_split
 
 import minst.utils as utils
 
+logger = logging.getLogger(__name__)
+
+
+class MissingDataException(Exception):
+    pass
+
 
 class Observation(object):
     """Document model each item in the collection."""
```

Comment:
Can we get a slightly better docstring with an example usage or explanation of the semantics?